.travis.yml

```diff
@@ -13,18 +13,15 @@ env:
   matrix:
     - PGVERSION=9.5
     - PGVERSION=9.6
+    - PGVERSION=10
 before_install:
-  - git clone -b v0.6.1 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b pg10_hack --depth 1 https://github.com/citusdata/tools.git
   - sudo make -C tools install
   - setup_apt
-  - curl https://install.citusdata.com/community/deb.sh | sudo bash
```

Comment:
What do you mean? I don't think we've released any 10-compatible extensions or packages thus far… Should I build some against `10beta1` or something?